File: DatosUsu.java

public class DatosUsu
{
   public static void main(String args[])
   {
      System.out.println("NOMBRE DE USUARIO: " +
                         System.getProperty("user.name"));
      System.out.println("HOME DIRECTORY: " +
                         System.getProperty("user.home"));
      System.out.println("DIRECTORIO DE TRABAJO: " +
                         System.getProperty("user.dir"));
   }
}

File: DatosUsu.cc
// DatosUsu.cc

#include <stdio.h>
#include <stdlib.h>

/* Despliega el contenido una variable de
   medio ambiente. */

int main()
{
  // Similar a "printenv USER".
  printf("NOMBRE DE USUARIO= %s\n", getenv("USER") );
  // Similar a "printenv HOME".
  printf("HOME DIRECTORY= %s\n", getenv("HOME") );
  // Similar a "printenv PWD".
  printf("DIRECTORIO DE TRABAJO= %s\n", getenv("PWD") );

  return 0;
}
